### ISSUE TITLE
Failed get accounts

### ISSUE THREAD

**Issue by Rysakov1986** [voiceip/tinyphone#14]
Hello!
If i have logged more (account1, account2) accounts, i can logout **account1** POST `/accounts/{{username}}@{{domain}}/logout`,
bur after rest POST {{host}}:6060/accounts return error http code 500:

```json
{
    "message": "Something went Wrong :("
}
```

log http:
```
(2020-10-29 16:49:39) [INFO    ] Request: 127.0.0.1:53731 012D0DF8 HTTP/1.1 GET /accounts
(2020-10-29 16:49:39) [INFO    ] Response: 012D0DF8 /accounts 500 0
```

log app:
```
2020-10-29 16:55:52.812    account.cpp !pjsua_acc_get_info(id, &pj_ai) error: Invalid value or argument (PJ_EINVAL) (status=70004) [..\src\pjsua2\account.cpp:1022]
```

and after try send login **account1** (POST `{{host}}:6060/login`)  return and crash app:
```json
{
    "account_name": "901@{{host}}",
    "id": 0,
    "message": "Account already exists, but login progress, please try again in sometime",
    "result": 202
}
```

if use logout all (POST `{{host}}:6060/logout`) no problem, all work very good!

Thx!

**Comment by kingster** [comment]
Thanks for reporting this issue. Will get this scenario checked and update here.

**Comment by kingster** [comment]
This has been resolved via #16 , please try with the latest [artifacts](https://ci.appveyor.com/project/kingster/tinyphone/builds/36048148/artifacts) and verify.
